README.rst

```diff
@@ -0,0 +1,300 @@
+"runnable recipes" addressing (1) where is the input data, (2) what software was
+used to analyse the data, (3) which computing environments were used to run the
+software and (4) which computational workflow steps were taken to run the
+analysis. This will permit to instantiate the analysis on the computational
+cloud and run the analysis to obtain (5) output results.
+
+
+1. Input data
+-------------
+The data being used is currently only available on request. It consists of a 10 GB ROOT file.
+
+2. Analysis code
+----------------
+
+You can either run the analysis step by step (explained below), or use the bash script :code:`run.sh` via:
+
+.. code-block:: console
+
+   $ bash run.sh <data_file> <tmp_phi_models> <result_plot>
+
+Step 0:
+
+`lhcbStyle.C <code/lhcbStyle.C>`_ -
+  Plot formats specific to the LHCb collaboration.
+
+
+`RooFitHeaders.h <code/RooFitHeaders.h>`_ -
+  Libraries and packages needed to perform the fitting.
+
+
+Step 1. `Optimise.C <code/Optimise.C>`_ -
+
+  Note: This is a very resource consuming process.
+
+  Optimisation is the process where combinations of different variable cuts are evaluated in order to maximise the signal yield and reduce the background. In this analysis, the optimisation study is performed to choose the combined BDT and particle identification (PID) selection criteria that maximise the expected statistical significance.
+  The result of the script is the heat map called `2D_Optimisation_Pi.pdf`. The higher significance (in yellow) means cleaner signal.
+  The optimal significance is found to be at the cuts of `BDT > 0.1` and `PIDmu > 2`.
+
+.. code-block:: console
+
+   $ root -b -q Optimisation.C(<data_file>)
+
+
+Step 2. `ModelFixing.C <code/ModelFixing.C>`_ -
+  Creates the theoretical model that the fit is compared to.
+
+.. code-block:: console
+
+   $ root -b -q ModelFixing.C(<data_file>, <tmp_phi_models>)
+
+Step 3. `OSMassFit.C <code/OSMassFit.C>`_ -
+  The fit for the signal was modelled with the sum of Crystal Ball distributions. Each shape consists of a Gaussian core with a power law tail on opposite sides. The background was modelled with a 2nd order Chebyshev polynomial distribution.
+
+.. code-block:: console
+
+   $ root -b -q OSMassFit.C(<data_file>, <tmp_phi_models>, <result_plot>)
+
+
+
+3. Compute environment
+----------------------
+In order to be able to rerun the analysis even several years in the future, we
+need to "encapsulate the current compute environment", for example to freeze the
+ROOT version our analysis is using. We shall achieve this by preparing a `Docker
+<https://www.docker.com/>`_ container image for our analysis steps.
+
+Some of the analysis steps will run in a pure `ROOT <https://root.cern.ch/>`_
+analysis environment. We can use an already existing container image, for
+example `reana-env-root6 <https://github.com/reanahub/reana-env-root6>`_, for
+these steps.
+
+Some of the other analysis tasks wil need ``hftools`` Python library installed
```

Comment:
Is it true that `hftools` is needed?  If we only need ROOT, then we can use a premade-container... that will make the example easier.

Reply:
Not actually needed, `legacy` from other `readme` file that I used as a template for this one.